### PR TITLE
[OLE32] stub for CoDisconnectContext function

### PR DESCRIPTION
**Pull request by Getequ**

### CODE REVIEW

File: dll/win32/ole32/compobj.c [hunk 1]
```diff
@@ -5165,6 +5165,16 @@ HRESULT WINAPI CoRegisterSurrogateEx(REFGUID guid, void *reserved)
     return E_NOTIMPL;
 }
 
+/***********************************************************************
+ *           CoDisconnectContext [OLE32.@]
+ */
+HRESULT WINAPI CoDisconnectContext(DWORD dwTimeout)
```
Comment (ThFabba) [inline dll/win32/ole32/compobj.c]: This file in Wine-synced, so these changes would need to be inside a `#ifdef __REACTOS__` block.
But even better, you can use `@ stdcall -stub CoDisconnectContext(long)` in the spec file to do this without any code change at all.

Comment (Getequ) [reply dll/win32/ole32/compobj.c]: If it's not critical i wish to leave it as S_OK stub

Comment (Getequ) [reply dll/win32/ole32/compobj.c]: done! I does not checked that stubs are being generating with returning 0, that equal to S_OK (i wished that). Thank you!

File: dll/win32/ole32/ole32.spec [hunk 2]
```diff
@@ -23,6 +23,7 @@
 # CoCreateObjectInContext
 # CoDeactivateObject
 @ stdcall -stub CoDisableCallCancellation(ptr)
+@ stdcall -stub CoDisconnectContext(long)
 @ stdcall CoDisconnectObject(ptr long)
 @ stdcall CoDosDateTimeToFileTime(long long ptr) kernel32.DosDateTimeToFileTime
```
Comment (HBelusca) [inline dll/win32/ole32/ole32.spec]: Why removing the kernel32 redirection?

**Comment by ThFabba** [comment]
So uhm, actually... Win2003 doesn't have this function, so it should only be added with `version=0x600` or similar

**Pull request merged**